chaoswm/actions.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+from logzero import logger
+
+from chaoslib.types import Configuration
+
+from .driver import Wiremock
+from .utils import get_wm_params, check_configuration
+
+__all__ = [
+    "add_mappings",
+    "populate_from_dir",
+    "delete_mappings",
+    "global_fixed_delay",
+    "global_random_delay",
+    "fixed_delay",
+    "random_delay",
+    "chunked_dribble_delay",
+    "down",
+    "up",
+    "reset"
+]
+
+
+def add_mappings(mappings: list = [], configuration: Configuration = {}):
```

Comment:
Two changes here:

1. Do not set default parameters to mutable values like `[]` or `{}`. Due to their mutable nature, this wil lead to very subtle bugs because that default value will be set once but changed inside the function. Next time it's called, it won't be empty anymore.

The right way of doing is as follows:

```
from typing import Any, List

def add_mappings(mappings: List[Any] = None, configuration: Configuration = None) -> List[Any]:
      mapping = mapping or []
      ....
  
```

2. The types should actually be imported from the builtin `typing`  module instead of using the functions `list` or `dict` which support the type protocol but not as clean.